injector/injector.go

```diff
@@ -244,7 +248,14 @@ func (m *Mutator) mutate(body []byte) ([]byte, error) {
 		})
 	}
 
-	return m.generateResponse(pod, admReviewReq, admReviewResponse, true, fmt.Sprintf("Mutation request for pod of marble type [%s] successful", marbleType))
+	msg := fmt.Sprintf("Mutation request for pod of marble type [%s] successful", marbleType)
+	resp, err := m.generateResponse(pod, admReviewReq, admReviewResponse, true, msg)
+	if err != nil {
+		return nil, err
```

Comment:
```suggestion
		m.log.Error("Unable to mutate request: response generation failed", zap.Error(err))
		return nil, err
```
shouldn't we log the error here?